kairon/api/app/routers/bot/data.py

```
@@ -369,6 +370,33 @@ async def upload_doc_content(
         event.enqueue()
     return {"message": "Document content upload in progress! Check logs."}
 
+@router.post("/upload/collection_data/{collection_name}", response_model=Response)
+async def upload_file_content(
+    file_content: UploadFile,
+    collection_name: str = Path(..., description="Collection name"),
+    overwrite: bool = False,
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.12.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)